### PR TITLE
Env setup

### PR DESCRIPTION
**Pull request by techlab-lainio**
- first draft for testing about docker-compose based full-stack env
- idea was to have very explicit and short samples..
- there can be bugs, please try to solve them to be able to have a full test round 🥇

### CODE REVIEW

File: tools/env/README.md [hunk 1]
```diff
@@ -19,14 +26,148 @@ The setup uses agency internal file ledger, intended only for testing during dev
...
+   alias cli=findy-agent-cli 
+   . <(findy-agent-cli completion bash | sed 's/findy-agent-cli/cli/g')
+   ```
+   Or if you have renamed the executable enter the following as well:
+   ```shell
+   export FCLI=<your-name-for-binary>
+   ```
+   That's for the helper scrips used in this directory.
+
+   **Admin Operations**
```
Comment (lauravuo-techlab) [inline tools/env/README.md:88]: This is excellent docs for admin capabilities but I think it is not the primary use case for most devs. Maybe we could duplicate this setup to CLI repository as such (without the wallet step), and craft here another use case between wallet and cli. And add a link to this cli setup from here.

File: tools/env/README.md [hunk 2]
```diff
@@ -19,14 +26,148 @@ The setup uses agency internal file ledger, intended only for testing during dev
...
+   development environment between restarts you should persist that key by
+   copying it to your environment variables. The key is in env `FCLI_KEY` after
+   running the setup script. The setup generates the `use-key.sh` script for
+   your convenient as well. Add `source use-key.sh` to your boot files for
+   example.
+
+   Next time you run the `./setup-cli-env.sh` it won't create a new key *if it
+   founds the existing one* i.e. you have sourced the `use-key.sh` script.
+
+   *Tip* Enter following commands:
```
[the remark this answers is not in the document]
Comment (lauravuo-techlab) [reply tools/env/README.md:77]: Oh, forget this, I think it was just a bug in setup-env-script.

File: tools/env/README.md [hunk 3]
```diff
@@ -19,14 +26,148 @@ The setup uses agency internal file ledger, intended only for testing during dev
...
+   To register your CLI authenticator for direct communication to Findy Agency
+   enter the following commands:
+   ```shell
+   source admin/register
+   source admin/login
+   ```
+   Later the login is all what is needed. After successful login you can enter
+   commands like:
+   ```shell 
+   cli agency count         # get status of the clould agents
```
Comment (lauravuo-techlab) [inline tools/env/README.md:107]: what if use hasn't renamed cli

File: tools/env/README.md [hunk 4]
```diff
@@ -19,14 +26,148 @@ The setup uses agency internal file ledger, intended only for testing during dev
...
+   cli agency loggin -L=5   # set login level of the core agency
```
Comment (lauravuo-techlab) [inline tools/env/README.md:108]: Typo, loggin -> logging

**Review by lauravuo-techlab — COMMENTED** [review]
Really good stuff! Everything worked for me except I skipped the cli renaming step -> couldn't copy-paste the commands directly. I would either make the cli alias step mandatory or use $FCLI etc. in the samples.

I have some comments on the content, but let's discuss online 👍

File: tools/env/setup-cli-env.sh [hunk 5]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ -z "$FCLI_KEY" ]; then
+	export FCLI_KEY=`cli new-key`
```
[the remark this answers is not in the document]
Comment (techlab-lainio) [reply tools/env/setup-cli-env.sh:4]: yes, there seem to be quite many cli naming bugs. The final idea was to use the same structure as e.g. `bob/login` script `cli=${FCLI:-findy-agent-cli}` but I forgot to change all the scripts to use that.

File: tools/env/README.md (hunk 1 above)
Comment (techlab-lainio) [reply tools/env/README.md:88]: good point 👍

File: tools/env/README.md (hunk 3 above)
Comment (techlab-lainio) [reply tools/env/README.md:107]: yes, this is the infamous cli naming bug.

**Pull request merged**